types/v8n.d.ts

```diff
@@ -0,0 +1,63 @@
+import { Rule, RuleDefinition } from './rule';
+import { ValidationError } from './validationerror';
+
+export interface V8nValidator {
+  chain: Rule[];
+  test(value?: any): boolean;
+  testAll(value?: any): ValidationError[];
+  check(value?: any): boolean;
+  testAsync(value?: any): boolean;
```

Comment:
It should return a `Promise<boolean>` instead.

Reply:
I fixed this :)